z390test/src/test/groovy/org/z390/test/TestOptables.groovy

```diff
@@ -703,4 +729,82 @@ class TestOptables extends z390Test {
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_z12.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_z12.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_z12.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_machine_zBC12() {
+        /**
+         * test 11D - machine(zBC12)
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_zBC12.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_zBC12.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_zBC12.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_machine_zEC12() {
+        /**
+         * test 11E - machine(zEC12)
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_zEC12.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_zEC12.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_zEC12.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_machine_zSeries6() {
+        /**
+         * test 11F - machine(zSeries-6)
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_zSeries-6.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_zSeries-6.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_zSeries-6.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_machine_ZS6() {
+        /**
+         * test 11G - machine(ZS-6)
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_ZS-6.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_ZS-6.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_ZS-6.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
+
+    @Test
+    void test_machine_ARCH10() {
+        /**
+         * test 11H - machine(ARCH-10)
+         */
+        var z390prn = basePath("rt", "mlc", "OPTB_ARCH-10.PRN")
+        env = ['Z390PRN': basePath("rt", "mlc", "OPTB_ARCH-10.PRN"),
+               'HLASMPRN': basePath("rt", "mlc", "OPTB#Z12.TF1")]
+        int rc = this.asmlg(basePath("rt", "mlc", "OPTB#"), *this.options, "sysprn(${z390prn})", "@${basePath("rt", "mlc", "OPTB_ARCH-10.OPT")}")
+        this.printOutput()
+        assert rc == 0
+    }
```

Comment:
Below the next brace is a red circle with a minus sign inside it. I think this means you hava a dangling line end character. Consider fixing this. Okay if you choose to not fix it.